packages/ui/src/components/BigInput.tsx

```diff
@@ -0,0 +1,177 @@
+              setShowAttachmentSheet(true);
+              editorRef.current?.editor?.blur();
+            }}
+          >
+            {uploadInfo?.imageAttachment && !uploadInfo.uploading ? (
+              <Image
+                source={{ uri: uploadInfo.imageAttachment }}
+                resizeMode="cover"
+                style={{
+                  width: '100%',
+                  height: '100%',
+                  borderBottomLeftRadius: 16,
+                  borderBottomRightRadius: 16,
+                }}
+              />
+            ) : (
+              <View
+                backgroundColor="$primaryText"
+                width="100%"
+                height="100%"
+                borderBottomLeftRadius="$xl"
+                borderBottomRightRadius="$xl"
+                padding="$2xl"
+                alignItems="center"
+                justifyContent="center"
+                gap="$l"
+              >
+                {uploadInfo?.uploading ? (
+                  <LoadingSpinner />
+                ) : (
+                  <Icon type="Camera" color="$background" />
+                )}
+              </View>
+            )}
+          </TouchableOpacity>
+          {channelType === 'notebook' && (
+            <View backgroundColor="$background" width="100%">
+              <Input
+                size="$xl"
+                height={titleInputHeight}
+                backgroundColor="$background"
+                borderColor="transparent"
+                placeholder="New Title"
+                onChangeText={setTitle}
+              />
+            </View>
+          )}
+        </View>
+      )}
+      <ScrollView
+        scrollEventThrottle={16}
+        scrollToOverflowEnabled
+        overScrollMode="always"
+        contentContainerStyle={{
+          paddingTop:
+            channelType === 'notebook'
+              ? titleInputHeight + imageButtonHeight
+              : 0,
+        }}
+      >
+        <MessageInput
+          shouldBlur={shouldBlur}
+          setShouldBlur={setShouldBlur}
+          send={send}
+          title={title}
+          image={uploadInfo?.uploadedImage ?? undefined}
+          channelId={channelId}
+          groupMembers={groupMembers}
+          storeDraft={storeDraft}
+          clearDraft={clearDraft}
+          getDraft={getDraft}
+          editingPost={editingPost}
+          setEditingPost={setEditingPost}
+          editPost={editPost}
+          setShowBigInput={setShowBigInput}
+          floatingActionButton
+          showAttachmentButton={false}
+          backgroundColor="$background"
+          paddingHorizontal="$m"
+          placeholder={placeholder}
+          bigInput
+          showToolbar={channelType === 'notebook'}
+          channelType={channelType}
+          ref={editorRef}
+        />
+      </ScrollView>
+      {channelType === 'notebook' &&
+        editorRef.current &&
+        editorRef.current.editor && (
+          <KeyboardAvoidingView
```

Comment:
Does this screen wrapper have anything in common with others we're using? I've been looking at trying to simplify how we handle them.

Reply:
I replaced the input for gallery rich text with this, not sure where else we would use it other than that though.